xapian-applications/omega/templates/query

```diff
@@ -86,7 +87,6 @@ $if{$not{$msizeexact},
 $list{$map{$queryterms,$list{$html{$uniq{$unstem{$_}}},<b>,</b>/<b>,</b>}:&nbsp;$nice{$freq{$_}}},Term frequencies: ,$. ,}
 <br><small>Search took $time seconds</small>
 <table>
-$if{$opt{clicklog},$set{queryid,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}}}
```

Comment:
What was the reason for moving this earlier?

It seems it was OK where it was to me, and it'll now get executed in cases when it'll never get used (for example when there's an error).